app/routes/rentals/show.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model(params) {
+    return this.store.queryRecord('rental', { title: params.title })
+  },
+
+  setupController(controller, model) {
```

Comment:
I thought it made the template more readable to access it as `rental` instead of `model` but I can revert it, nbd.  Is it considered bad practice to use `setupController` in this way?